frontend/packages/ux-editor/src/containers/EditContainer.tsx

```diff
@@ -219,13 +221,15 @@ export function EditContainer(props: IEditContainerProps) {
       {isEditMode && (
         <div className={classes.buttons}>
           <Button
+            aria-label={t('general.cancel')}
```

Comment:
Kanskje også legge til den samme teksten i et `title`-attributt? (Gjelder alle disse knappene.)

Reply:
Nice, we could set the title attribute on the SVG, and then I can remove the aria label because the title will bobble up to be the accessibility name for the button. :)